### ISSUE TITLE
Accessibility: dark mode text fails WCAG contrast minimums

### ISSUE THREAD

**Issue by calebfoss** [processing/p5.js-website#687]
### Most appropriate sections of the p5.js website?

Reference

### What is your operating system?

None

### Actual Behavior

![Blue text on a dark gray background with extremely low contrast making it difficult to read](https://github.com/user-attachments/assets/258b8fc7-3cd2-4947-bb67-0a89fd2cc2bf)

The text in the Syntax section of references pages is being inverted, so it's `#2012CC` on `#212121` with an [extremely low contrast ratio of 1.54:1, which fails all WCAG standards.](https://webaim.org/resources/contrastchecker/?fcolor=2012CC&bcolor=212121)

![Black text on a dark gray background with extremely low contrast making it difficult to read](https://github.com/user-attachments/assets/54bd6603-cbf6-4aa5-a487-17f95316f5d8)

In monochrome dark mode, [the ratio is even lower at 1.3:1](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=212121).

### Expected Behavior

Colors selected that meet accessible contrast minimums. 

### Would you like to work on the issue?

No

**Comment by davepagurek** [comment]
Thanks @calebfoss! This definitely needs addressing, and we have someone looking into it in https://github.com/processing/p5.js-website/issues/680, so I'm going to close this one as a duplicate.
